doc/sphinx/Clusters_from_Scratch/active-active.rst

```diff
@@ -29,28 +29,62 @@ Distributed Lock Manager (DLM) required by cluster filesystems:
 
     # yum install -y gfs2-utils dlm
 
+.. NOTE::
+
+    Because of `an open CentOS bug <https://bugs.centos.org/view.php?id=16939>`,
```

Comment:
This needs the easy-to-forget underbar at the end to make it clickable (ditto for the next chapter)